linux/build.sh

```diff
@@ -0,0 +1,3 @@
+conda create -p /tmp/snakestagram --copy -y -q python=3 pandas scikit-learn ipykernel
+
+tar czf /tmp/snakestagram-v$VERSION-linux-x64.tar.gz /tmp/snakestagram
```

Comment:
While you're at it, I would `cd` to `/tmp` and make this tarball up `./snakestagram` so it doesn't nest `/tmp`